File: Golang/2.BonusCoklat/2.BonusCoklat.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var Jumlah int
	var bonusTukar int
	var Total int

	fmt.Printf("Jumlah Bungkus Coklat yang ditukar : ")
	fmt.Scanf("%d", &Jumlah)
	Nilai := math.Floor(float64(Jumlah)) / 5
	fmt.Println(math.Floor(float64(Jumlah)) / 5)

	fmt.Println(Nilai)
	if Nilai > 5 {
		bonusTukar = int(Nilai) / 5
	} else {
		bonusTukar = 0
	}
	fmt.Println(bonusTukar)

	Total = Jumlah + int(Nilai) + int(bonusTukar)

	fmt.Println("Your value feedback: ")
	fmt.Println(Total)
}

File: Javascript/2.BonusCoklat.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function BonusCoklat(Jumlah){
    let Nilai = Math.floor(Jumlah);
    let bonusTukar = Nilai / 5;
    let totalAll, tukarBaru;
    
    if(Math.floor(bonusTukar) > 5){
        tukarBaru = bonusTukar / 5;
    }else{
        tukarBaru = 0;
    }

    totalAll = Nilai + bonusTukar + tukarBaru;

    return Math.floor(totalAll);
}

rl.question('Jumlah Bungkus Coklat yang ditukar: ', (answer) => {
    let result = BonusCoklat(answer);
    console.log(`Thank you for your valuable feedback: ${result} `);
    BonusCoklat(answer);
    rl.close();
});
